### PR TITLE
Removes queen sweep.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Queen never needed sweep, it was added as a way to allow the caste to frontline with nearly no punishment like a real solo gamer that was literally a raidboss.
This is bad for the game for several reasons as the caste then has literally the best stats in the game overall, an amazing stun button and the best pheromones.
Queen has been overbuffed in general. Which is severely dumb. This button should go.

## Why It's Good For The Game
This button should've never been added to queen. She shouldn't benefit from being cqc. If you're alone without a hive you should die. Period.

## Changelog
:cl:
balance: Queen no longer has a tail sweep.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -42. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Marmio64** [comment]
on one hand, queen should always be with their hive, on the other, queen SHOULD be the most powerful caste.

**Comment by DerFlammenwerfer** [comment]
> on one hand, queen should always be with their hive, on the other, queen SHOULD be the most powerful caste.

They already are, even without the sweep. The screech is all the stunning power they should have. All the sweep is at this stage is a get out of jail free card to be used when they overextend and are getting bodyblocked for doing so.

**Comment by qfmysteryman23** [comment]
This was a horrible thing to remove from the queen and the fact that this was merged while having far more down votes then up votes is despicable.

**Comment by Al-1ce** [comment]
>inb4 "muh amazing stun"
the screech is terrible in any practical use, --people hype it up to be that a queen can just waltz in the FOB and screech to instantly win. It's okay for getting one and maybe even two caps if you have a **coordinated team**, but when does that happen?
>the most powerful caste shouldn't be a frontline combatant.
...yes, and no. That defeats the entire purpose of the queen, likewise the stats are stupidly high for a "support" role. 
>b-but my support!!!
The queen should focus on the hive, sure, that's a given. --That doesn't apply as much when you've got a shrike or few drones taking care of that, or simply don't have any captures yet. The tail sweep should have a longer cooldown to balance out what is already an incredibly robust caste.

tl;dr give the tailsweep a longer cooldown.

**Comment by saprasam** [comment]
if there was ever a port of TG crawling you could always make it so queen tailsweep just knocks instead of hardstunning

**Review by Hughgent — CHANGES_REQUESTED** [review]
Remove the ability, but keep the code for the tail sweep. Someone might want to add it later, or add it to another xenomorph.

**Comment by Fikou** [comment]
> Remove the ability, but keep the code for the tail sweep. Someone might want to add it later, or add it to another xenomorph.

defenders already have tail sweep

**Comment by Dolth** [comment]
I think the queen should keep the tailsweep since it's still meant to be the strongest unique caste. Although to prevent it from being used offensively it can be tweaked into a longer cool down.

**Review by TiviPlus — APPROVED** [review]
Someone just copypasted the exact same code twice in two places, code is good, will have to think on balance

**Comment by MiniMeatwad** [comment]
Make the delay longer so it's not viable offensively.

**Comment by Texicos** [comment]
Why the fuck did you merge this when almost everyone who looked and it down voted

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-50). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.
